<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tcp_FileServer_Recv</class>
 <widget class="QMainWindow" name="Tcp_FileServer_Recv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>968</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>接收文件</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="serverStatusLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>服务端就绪</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>接收</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="clientProgressBar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="serverProgressBar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1366</width>
      <height>768</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ip">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.104</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
